openassessment/workflow/models.py

```diff
@@ -111,6 +108,12 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super(AssessmentWorkflow, self).__init__(*args, **kwargs)
+
```

Comment:
Same as the comment above.

Reply:
The Assessment Score Priority was initialized at the class level by obtaining value of `ORA2_ASSESSMENT_SCORE_PRIORITY` variable from settings module . This is the right approach if we know that the Assessment Score Priority wouldn't be updated/changed during runtime but during tests we need to check the code with settings patched/updated at the runtime so that would only be possible if the Assessment Score Priority is initialized inside the constructor i.e initialized at the runtime.

Reply:
@ahmedbilal Can you revert these changes? 
I took a deeper look at your code and noticed that you were trying to import a Django model inside the XBlock, and it can't be done that way.

Django methods/models/etc can't be accessed inside an XBlock, and that's why we have multiple APIs that are in turn used by the XBlock.

For this ticket, just consider the platform's default priority when grading, but add a note to the PR explicitly stating it.

Can you revert these changes to the `models.py` given what I've said above?